### PR TITLE
ROX-18443: Remove tech-preview warning for NP-Guard integrated code

### PR DESCRIPTION
**Pull request by vikin91**
## Description

As in the title.

## Checklist
- [x] Investigated and inspected CI test results

### N/A
- [ ] Unit test and regression tests added
- [ ] Evaluated and added CHANGELOG entry if required
- [ ] Determined and documented upgrade steps
- [ ] Documented user facing changes (create PR based on [openshift/openshift-docs](https://github.com/openshift/openshift-docs) and merge into [rhacs-docs](https://github.com/openshift/openshift-docs/tree/rhacs-docs))

If any of these don't apply, please comment below.

### Here I tell how I validated my change

- Running the commands locally and making sure that the tech preview warning is not visible
- Running the commands with `-h` argument and checking that help texts do not contain any tech preview text

### Reminder for reviewers

In addition to reviewing code here, reviewers **must** also review testing and request further testing in case the
performed one does not seem sufficient. As a reviewer, you must not approve the change until you understand the
performed testing and you are satisfied with it.

### CODE REVIEW

File: roxctl/connectivity-map/command.go [hunk 1]
```diff
@@ -13,7 +12,7 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	c := &cobra.Command{
 		Use:   "connectivity-map <folder-path>",
 		Short: "(Technology Preview) Analyze connectivity based on network policies and other resources.",
```
Comment (zivnevo) [inline roxctl/connectivity-map/command.go:14]: Shouldn't `(Technology Preview)` be removed from the `Short` texts as well?

Comment (vikin91) [reply roxctl/connectivity-map/command.go:14]: Very good catch!

File: roxctl/generate/generate.go [hunk 2]
```diff
@@ -12,7 +11,7 @@ func Command(cliEnvironment environment.Environment) *cobra.Command {
 	c := &cobra.Command{
 		Use:   "generate",
 		Short: "(Technology Preview) Commands related to generating different resources.",
```
Comment (Maddosaurus) [inline roxctl/generate/generate.go:13]:
```suggestion
		Short: "Commands related to generating different resources.",
```

I think we might have another Technology Preview we can remove, correct?

**Review by Maddosaurus — CHANGES_REQUESTED** [review]
Good to go, besides the one more potential Short Command update. CI looks good as well afaik

**Review by Maddosaurus — APPROVED** [review]
Let me switch the request changes to an Approve/Nit, as I just saw the `generate` command is going away in another PR :smile:

File: roxctl/generate/generate.go (hunk 2 above)
Comment (vikin91) [reply roxctl/generate/generate.go:13]: Thanks! I found two more :) Both will be removed in the other PR. To save time, let me merge this and I will double check in the next PR.

**Pull request merged**